nextcord/ui/view.py

```diff
@@ -505,16 +505,17 @@ def __init__(self, state: ConnectionState):
         self._synced_message_views: Dict[int, View] = {}
         self._state: ConnectionState = state
 
-    @property
-    def persistent_views(self) -> Sequence[View]:
-        # fmt: off
-        views = {
-            view.id: view
-            for (_, (view, _)) in self._views.items()
-            if view.is_persistent()
-        }
-        # fmt: on
-        return list(views.values())
+    def views(self, persistent: bool = True) -> Sequence[View]:
+        views = [view for (view, _) in self._views.values()]
+
+        # I'm not sure if there is a better way to do this
+        check: Callable[[View], bool]
+        if persistent:
+            check = lambda v: v.is_persistent()
+        else:
+            check = lambda v: not v.is_persistent()
+
+        return list(filter(check, views))
```

Comment:
So either you get all persistent views, or all non persistent views. Is there a way to just get all views?

Reply:
fa93c626